var/spack/repos/builtin/packages/gcc/package.py

```diff
@@ -157,6 +159,9 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     depends_on("automake@1.15.1:", when="@master", type="build")
     depends_on("autoconf@2.69:", when="@master", type="build")
 
+    depends_on("gmake@3.80:")
+    depends_on("perl@5")
```

Comment:
Isn't perl just a build dependency? According to Fedora (https://src.fedoraproject.org/rpms/gcc/blob/ca788bfdffb0f4bd7b210e848f2007cec8829de9/f/gcc.spec#_266), make seems to be a build and run dependency.

Reply:
Thanks, forgot to add dep types :slightly_smiling_face:

Reply:
Hm, is make really a runtime dep? I thought the LTO stuff just generates a temporary makefile if it detects it runs under make's jobserver, and then it invokes the make set in an env variable. So, if that's the case it's not really a dependency

Reply:
I went for just build deps, we can check whether `gmake `is a run dep later and fix the recipe in a following PR in case